src/utils/proxy.ts

```diff
@@ -49,7 +50,10 @@ export async function proxyRequest(
 
   // Headers
   const fetchHeaders = mergeHeaders(
-    getProxyRequestHeaders(event, { host: target.startsWith("/") }),
+    getProxyRequestHeaders(event, {
+      host: target.startsWith("/"),
+      trustedHeaders: opts.trustedHeaders,
```

Comment:
I think we can call it `proxyHeaders` or `includeHeaders` (can be coupled with another `excludeHeader`). They are not trusted only being passed.

Reply:
I once named it `forwardHeaders`; perhaps `includeHeaders` would be bettet, as you said

Reply:
I like `forwardHeaders`!

Reply:
Done 🙌